droidlet/tools/crowdsourcing/turk_as_oncall/server_files/oncall_task.html

```diff
@@ -0,0 +1,206 @@
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>	
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>	
+<!-- Modal --><!-- MODAL -->	
+<div	
+  aria-hidden="true"	
+  aria-labelledby="exampleModalLabel"	
+  class="modal fade bd-example-modal-lg"	
+  id="exampleModal"	
+  role="dialog"	
+  tabindex="-1"	
+>	
+  <div class="modal-dialog modal-lg" role="document">	
+    <div class="modal-content">	
+      <div class="modal-header">	
+        <nav aria-label="Instruction page navigation">	
+          <ul class="pagination justify-content-center">	
+            <li class="page-item disabled" id="previous">	
+              <a class="page-link" tabindex="-1" href="#" onclick="incrementPage(-1);return false;" role="button" aria-expanded="false" aria-controls="collapseExample">Previous</a>	
+            </li>	
+            <li class="page-item active" id="pg1">	
+              <a class="page-link" tabindex="-1" href="#" onclick="applyPagination(1);return false;" role="button" aria-expanded="true" aria-controls="collapseExample">1</a>	
+            </li>	
+            <li class="page-item" id="pg2">	
+              <a class="page-link" tabindex="-1" href="#" onclick="applyPagination(2);return false;" role="button" aria-expanded="false" aria-controls="collapseExample">2</a>	
+            </li>	
+            <li class="page-item" id="pg3">	
+              <a class="page-link" tabindex="-1" href="#" onclick="applyPagination(3);return false;" role="button" aria-expanded="false" aria-controls="collapseExample">3</a>	
+            </li>	
+            <li class="page-item" id="next">	
+              <a class="page-link" tabindex="-1" href="#" onclick="incrementPage(1);return false;" role="button" aria-expanded="false" aria-controls="collapseExample">Next</a>	
+            </li>	
+          </ul>	
+        </nav>	
+      </div>	
+      <div class="modal-body" id="modal"></div>	
+      <div class="modal-footer">	
+        <button class="btn btn-primary" id="close-instructions-btn" data-dismiss="modal" type="button" style="display:none">Close</button>
+      </div>	
+    </div>	
+  </div>	
+</div>
+
+	<!-- Content -->	
+  <section	
+  class="container"	
+  id="Other"	
+  style="	
+    padding: 10px;	
+    font-family: Verdana, Geneva, sans-serif;	
+    font-size: 0.9em;	
+  "	
+>	
+  <div class="row col-xs-12 col-md-12">	
+    <!-- Instructions -->	
+    <div class="panel panel-primary">	
+      <div class="panel-heading">	
+        <button class="btn btn-primary" type="button" onclick="showHideInstructions()" aria-expanded="false" aria-controls="instructionsWrapper">	
+          Click here to view/hide instructions	
+        </button>	
+      </div>	
+      <div class="collapse" id=instructionsWrapper>	
+        <div id="instructions">	
+          <div class="collapse in" id="instructions-page-1">	
+            <h1><strong>Help find bugs in an interactive assistant</strong></h1>    	
```

Comment:
nit: rephrase "Help find bugs in an interactive assistant" -> "Interact with an assistant and mark bugs on failure" ?